### PR TITLE
collation: improve some new_collation_enabled relateds doc

### PR DESCRIPTION
**Pull request by Defined2014**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): ref https://github.com/pingcap/tidb/issues/46136

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: mysql-schema.md [hunk 1]
```diff
@@ -69,7 +69,11 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
 ## 其它系统表
 
 * `GLOBAL_VARIABLES` 全局系统变量表
-* `tidb` 用于 TiDB 在 bootstrap 的时候记录相关版本信息
+* `tidb` 用于记录 TiDB 的一些全局信息
+    * `bootstrapped` 用于记录 TiDB 是否已经 bootstrap 完成
+    * `tidb_server_version` 用于记录 TiDB 在 bootstrap 时候的版本信息
+    * `system_tz` 用于记录 TiDB 的系统时区
+    * `new_collation_enabled` 用于记录 TiDB 是否开启了 [新排序规则框架](/character-set-and-collation.md#新框架下的排序规则支持)
```
Comment (bb7133) [inline mysql-schema.md:76]:
```suggestion
    * `new_collation_enabled` 用于记录 TiDB 是否开启了 [新排序规则框架](/character-set-and-collation.md#新框架下的排序规则支持)，注意该值为只读，不可修改。
```

Comment (bb7133) [reply mysql-schema.md]: 我感觉 `tidb` 这张表移到 '其它系统表' 外面比较好，比如叫做 '集群状态系统表'。

Comment (Defined2014) [reply mysql-schema.md]: Done

File: br/backup-and-restore-overview.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
 |  ----  | ----  | ----- |
```
Comment (ran-huang) [inline br/backup-and-restore-overview.md:115]: Which version do these changes apply to?

Comment (Defined2014) [reply br/backup-and-restore-overview.md:115]: I think always been like this

Comment (ran-huang) [reply br/backup-and-restore-overview.md:115]: OK. Then we should cherry-pick it to at least the latest LTS.

File: mysql-schema.md [hunk 3]
```diff
@@ -22,6 +22,14 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
 * `global_priv` 基于证书的认证信息
 * `role_edges` 角色之间的关系信息
 
+## 集群状态系统表
+
+* `tidb` 用于记录 TiDB 的一些全局信息
+    * `bootstrapped` 用于记录 TiDB 是否已经 bootstrap 完成，注意该值为只读，不可修改。
+    * `tidb_server_version` 用于记录 TiDB 在 bootstrap 时候的版本信息，注意该值为只读，不可修改。
+    * `system_tz` 用于记录 TiDB 的系统时区
+    * `new_collation_enabled` 用于记录 TiDB 是否开启了 [新排序规则框架](/character-set-and-collation.md#新框架下的排序规则支持)，注意该值为只读，不可修改。
```
Comment (ran-huang) [inline mysql-schema.md:31]:
```suggestion
    * `new_collation_enabled` 用于记录 TiDB 是否开启了[新排序规则框架](/character-set-and-collation.md#新框架下的排序规则支持)，注意该值为只读，不可修改。
```

File: mysql-schema.md [hunk 4]
```diff
@@ -22,6 +22,14 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
 * `global_priv` 基于证书的认证信息
 * `role_edges` 角色之间的关系信息
 
+## 集群状态系统表
+
+* `tidb` 用于记录 TiDB 的一些全局信息
```
Comment (ran-huang) [inline mysql-schema.md:27]:
```suggestion
* `tidb` 用于记录 TiDB 的一些全局信息

```

**Review by ran-huang — APPROVED** [review]

File: br/backup-and-restore-overview.md [hunk 5]
```diff
@@ -115,7 +115,7 @@ TiDB 支持将数据备份到 Amazon S3、Google Cloud Storage (GCS)、Azure Blo
...
 |GBK charset|| BR 在 v5.4.0 之前不支持恢复 `charset=GBK` 的表。并且，任何版本的 BR 都不支持恢复 `charset=GBK` 的表到 v5.4.0 之前的 TiDB 集群。|
 | 聚簇索引 | [#565](https://github.com/pingcap/br/issues/565)       | 确保恢复时集群的 `tidb_enable_clustered_index` 全局变量和备份时一致，否则会导致数据不一致的问题，例如 `default not found` 和数据索引不一致。 |
-| New collation  | [#352](https://github.com/pingcap/br/issues/352)       | 确保恢复时集群的 `new_collations_enabled_on_first_bootstrap` 变量值和备份时的一致，否则会导致数据索引不一致和 checksum 通不过。更多信息，请参考 [FAQ - BR 为什么会报 `new_collations_enabled_on_first_bootstrap` 不匹配？](/faq/backup-and-restore-faq.md#恢复时为什么会报-new_collations_enabled_on_first_bootstrap-不匹配)。 |
+| New collation  | [#352](https://github.com/pingcap/br/issues/352)       | 确保恢复时集群的 `mysql`.`tidb` 表中 `new_collation_enabled` 变量值和备份时的一致，否则会导致数据索引不一致和 checksum 通不过。更多信息，请参考 [FAQ - BR 为什么会报 `new_collations_enabled_on_first_bootstrap` 不匹配？](/faq/backup-and-restore-faq.md#恢复时为什么会报-new_collation_enabled-不匹配)。 |
```
Comment (qiancai) [inline br/backup-and-restore-overview.md:118]:
```suggestion
| New collation  | [#352](https://github.com/pingcap/br/issues/352)       | 确保恢复时集群的 `mysql.tidb` 表中 `new_collation_enabled` 变量值和备份时的一致，否则会导致数据索引不一致和 checksum 通不过。更多信息，请参考 [FAQ - BR 为什么会报 `new_collations_enabled_on_first_bootstrap` 不匹配？](/faq/backup-and-restore-faq.md#恢复时为什么会报-new_collation_enabled-不匹配)。 |
```

File: character-set-and-collation.md [hunk 6]
```diff
@@ -484,7 +484,13 @@ Query OK, 1 row affected
 
 ### 新框架下的排序规则支持
 
-TiDB 4.0 新增了完整的排序规则支持框架，从语义上支持了排序规则，并新增了配置开关 `new_collations_enabled_on_first_bootstrap`，在集群初次初始化时决定是否启用新排序规则框架。如需启用新排序规则框架，可将 `new_collations_enabled_on_first_bootstrap` 的值设为 `true`，详情参见 [`new_collations_enabled_on_first_bootstrap`](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap)。要在该配置开关打开之后初始化集群，可以通过 `mysql`.`tidb` 表中的 `new_collation_enabled` 变量确认是否启用了新排序规则框架：
+TiDB 4.0 新增了完整的排序规则支持框架，从语义上支持了排序规则，并新增了配置开关 `new_collations_enabled_on_first_bootstrap`，在集群初次初始化时决定是否启用新排序规则框架。如需启用新排序规则框架，可将 `new_collations_enabled_on_first_bootstrap` 的值设为 `true`，详情参见 [`new_collations_enabled_on_first_bootstrap`](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap)。
+
+对于一个已经初始化完成的 TiDB 集群，可以通过 `mysql`.`tidb` 表中的 `new_collation_enabled` 变量确认是否启用了新排序规则框架。
```
Comment (qiancai) [inline character-set-and-collation.md:489]:
```suggestion
对于一个已经初始化完成的 TiDB 集群，可以通过 `mysql.tidb` 表中的 `new_collation_enabled` 变量确认是否启用了新排序规则框架。
```

File: character-set-and-collation.md [hunk 7]
```diff
@@ -484,7 +484,13 @@ Query OK, 1 row affected
...
+
+> **注意：**
+>
+> 当 `mysql`.`tidb` 表查询结果和 `new_collations_enabled_on_first_bootstrap` 的值不同，以 `mysql`.`tidb` 表结果为准。
```
Comment (qiancai) [inline character-set-and-collation.md:493]:
```suggestion
> 当 `mysql.tidb` 表查询结果和 `new_collations_enabled_on_first_bootstrap` 的值不同时，以 `mysql.tidb` 表的结果为准。
```

File: faq/backup-and-restore-faq.md [hunk 8]
```diff
@@ -133,9 +133,9 @@ Error: failed to check gc safePoint, checkpoint ts 433177834291200000: GC safepo
 
 TiCDC 可以通过配置项中的 [`filter.rules`](https://github.com/pingcap/tiflow/blob/7c3c2336f98153326912f3cf6ea2fbb7bcc4a20c/cmd/changefeed.toml#L16) 项完成，Drainer 则可以通过 [`syncer.ignore-table`](/tidb-binlog/tidb-binlog-configuration-file.md#ignore-table) 完成。
 
-### 恢复时为什么会报 `new_collations_enabled_on_first_bootstrap` 不匹配？
+### 恢复时为什么会报 `new_collation_enabled` 不匹配？
 
-从 TiDB v6.0.0 版本开始，[`new_collations_enabled_on_first_bootstrap`](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) 配置项的默认值由 `false` 改为 `true`。BR 会备份上游集群的 `new_collations_enabled_on_first_bootstrap` 配置项。当上下游集群的此项配置相同时，BR 才会将上游集群的备份数据安全地恢复到下游集群中。若上下游的该配置不相同，BR 会拒绝恢复，并报此配置项不匹配的错误。
+从 TiDB v6.0.0 版本开始，[`new_collations_enabled_on_first_bootstrap`](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) 配置项的默认值由 `false` 改为 `true`。BR 会备份上游集群的 `mysql`.`tidb` 表中的 `new_collation_enabled` 配置项。当上下游集群的此项配置相同时，BR 才会将上游集群的备份数据安全地恢复到下游集群中。若上下游的该配置不相同，BR 会拒绝恢复，并报此配置项不匹配的错误。
```
Comment (qiancai) [inline faq/backup-and-restore-faq.md:138]:
```suggestion
从 TiDB v6.0.0 版本开始，[`new_collations_enabled_on_first_bootstrap`](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap) 配置项的默认值由 `false` 改为 `true`。BR 会备份上游集群的 `mysql.tidb` 表中的 `new_collation_enabled` 配置项。当上下游集群的此项配置相同时，BR 才会将上游集群的备份数据安全地恢复到下游集群中。若上下游的该配置不相同，BR 会拒绝恢复，并报此配置项不匹配的错误。
```

File: migration-tools.md [hunk 9]
```diff
@@ -52,7 +52,7 @@ TiDB 提供了丰富的数据迁移相关的工具，用于全量迁移、增量
 | **上游** | TiDB |
 | **下游（输出文件）** | SST，backup.meta 文件，backup.lock 文件 |
 | **主要优势** | <ul><li>适用于向另一个 TiDB 迁移数据。</li><li>支持数据冷备份到外部存储，可以用于灾备恢复。</li></ul> |
-| **使用限制** | <ul><li>BR 恢复到 TiCDC / Drainer 的上游集群时，恢复数据无法由 TiCDC / Drainer 同步到下游。</li><li>BR 只支持在 `new_collations_enabled_on_first_bootstrap` 开关值相同的集群之间进行操作。</li></ul> |
+| **使用限制** | <ul><li>BR 恢复到 TiCDC / Drainer 的上游集群时，恢复数据无法由 TiCDC / Drainer 同步到下游。</li><li>BR 只支持在 `mysql`.`tidb` 表中 `new_collation_enabled` 开关值相同的集群之间进行操作。</li></ul> |
```
Comment (qiancai) [inline migration-tools.md:55]:
```suggestion
| **使用限制** | <ul><li>BR 恢复到 TiCDC / Drainer 的上游集群时，恢复数据无法由 TiCDC / Drainer 同步到下游。</li><li>BR 只支持在 `mysql.tidb` 表中 `new_collation_enabled` 开关值相同的集群之间进行操作。</li></ul> |
```

File: mysql-schema.md [hunk 10]
```diff
@@ -22,6 +22,15 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
...
+
+    * `bootstrapped` 用于记录 TiDB 是否已经 bootstrap 完成，注意该值为只读，不可修改。
```
Comment (qiancai) [inline mysql-schema.md:29]:
```suggestion
    * `bootstrapped` 用于记录 TiDB 集群是否已完成初始化，注意该值为只读，不可修改。
```

File: mysql-schema.md [hunk 11]
```diff
@@ -22,6 +22,15 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
...
+    * `tidb_server_version` 用于记录 TiDB 在 bootstrap 时候的版本信息，注意该值为只读，不可修改。
```
Comment (qiancai) [inline mysql-schema.md:30]:
```suggestion
    * `tidb_server_version` 用于记录 TiDB 在初始化时的版本信息，注意该值为只读，不可修改。
```

**Comment by qiancai** [comment]
Removed the `needs-cherry-pick-release-7.2` label because the v7.2 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.2 and will no longer receive new updates.

**Review by qiancai — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**